Xamarin.Forms.Platform.Android/AppCompat/FormsAppCompatActivity.cs

```diff
@@ -126,11 +126,13 @@ protected void LoadApplication(Application application)
 
 			application.PropertyChanged += AppOnPropertyChanged;
 
-			var iver = Platform.GetRenderer(application.MainPage);
-			if (iver != null)
+			if (application != null) 
```

Comment:
Shouldn't the event subscription in L127 be also after the check?

Reply:
Yes

Reply:
Huh, L127 that was there before the fix to 59882: https://github.com/xamarin/Xamarin.Forms/pull/1212/files

Reply:
Actually there is already a null check in lines 116-117 so either the issue is a bit more intricate or the fix is unnecessary. Or I'm not seeing something at 10:30PM

Reply:
Please see updated Bugzilla comment for actual issue.